espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/Handlers/Finder.java

```diff
@@ -13,6 +13,7 @@
 import io.appium.espressoserver.lib.Exceptions.InvalidStrategyException;
 import io.appium.espressoserver.lib.Exceptions.ServerErrorException;
 import io.appium.espressoserver.lib.Http.Response.AppiumResponse;
+import io.appium.espressoserver.lib.Model.Appium;
```

Comment:
Appium is the serializable model and AppiumResponse is the HTTP response.

What I could do is rename Appium to AppiumResponse and then just change AppiumResponse to BaseResponse since _all_ responses have the same format.